MineColab.ipynb

```diff
@@ -58,16 +58,16 @@
         "from pyngrok import conf, ngrok\n",
         "# Update the package lists\n",
         "!sudo apt update &>/dev/null && echo \"apt cache successfully updated\" || echo \"apt cache update failed, you might receive stale packages\"\n",
-        "# Install OpenJDK 15\n",
+        "# Install OpenJDK 16\n",
         "!wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -\n",
-        "!sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ &>/dev/null || echo \"Failed to add repo. Still can be ignored if openjdk15 gets installed.\"\n",
-        "!sudo apt-get install adoptopenjdk-15-hotspot-jre &>/dev/null && echo \"Yay! Openjdk15 has been successfully installed.\" || echo \"Failed to install OpenJdk15.\"\n",
+        "!sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ &>/dev/null || echo \"Failed to add repo. Still can be ignored if openjdk16 gets installed.\"\n",
+        "!sudo apt-get install adoptopenjdk-16-hotspot-openj9 &>/dev/null && echo \"Yay! Openjdk16 has been successfully installed.\" || echo \"Failed to install OpenJdk16.\"\n",
```

Comment:
Umm...there's no jvm named `adoptopenjdk-16-hotspot-openj9`. Both hotspot and openj9 exist as independent jvms.

Reply:
It should be `adoptopenjdk-16-openj9`, propably

Reply:
> Umm...there's no jvm named `adoptopenjdk-16-hotspot-openj9`. Both hotspot and openj9 exist as independent jvms.
> Maybe a typo while making the pr.
 
oops